Documentation/FAQ.md

```diff
@@ -28,11 +28,15 @@ The SerenityOS project tries to maximize hackability, accountability, and fun(!)
 
 ## Does SerenityOS have a package manager?
 
-No, SerenityOS does not have a package manager. The project uses a monorepo approach, meaning that all software is built in the same style and using the same tools. There is no reason to have something like a package manager because of this.
+To simplify: No, SerenityOS does not have a "Linux-style" package manager with pre-built software. The `pkg` tool interacts with the existing SerenityOS Ports system.
```

Comment:
I don't like mentioning the pkg tool and the Port system before the paragraph with *However*. it seems confusing to reference a concept before introducing it.

I also don't think the "To simplify:" clause makes sense. Perhaps "In short, " ? or just omit

Reply:
I'll omit pgk here; whoever wants to read on will find out about pkg and the Ports soon enough.